Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII_test/TestCommands/collection_example.xml

```diff
@@ -0,0 +1,284 @@
+                <stix:STIX_Header>
+                    <stix:Handling>
+                        <marking:Marking>
+                            <marking:Controlled_Structure>../../../../descendant-or-self::node()</marking:Controlled_Structure>
+                            <marking:Marking_Structure xsi:type="tlpMarking:TLPMarkingStructureType" color="WHITE"/>
+                            <marking:Marking_Structure xsi:type="TOUMarking:TermsOfUseMarkingStructureType">
+                                <TOUMarking:Terms_Of_Use>TBD</TOUMarking:Terms_Of_Use>
+                            </marking:Marking_Structure>
+                            <marking:Marking_Structure xsi:type="simpleMarking:SimpleMarkingStructureType">
+                                <simpleMarking:Statement>Unclassified (Public)</simpleMarking:Statement>
+                            </marking:Marking_Structure>
+                        </marking:Marking>
+                    </stix:Handling>
+                </stix:STIX_Header>
+                <stix:Observables cybox_major_version="2" cybox_minor_version="1" cybox_update_version="0">
+                    <cybox:Observable id="opensource:Observable-cfe14953-f986-450b-93f1-0041a597b8d2" sighting_count="1">
+                        <cybox:Title>URL: http://example.org/...</cybox:Title>
+                        <cybox:Description>URL: http://example.org/| isOnline:yes| dateVerified:2021-10-19T10:07:34+00:00</cybox:Description>
+                        <cybox:Object id="opensource:URI-fa125422-3419-4d68-8a78-fd6a5c5f2dec">
+                            <cybox:Properties xsi:type="URIObj:URIObjectType" type="URL">
+                                <URIObj:Value condition="Equals">http://example.org/</URIObj:Value>
+                            </cybox:Properties>
+                        </cybox:Object>
+                    </cybox:Observable>
+                </stix:Observables>
+            </stix:STIX_Package>
+        </taxii_11:Content>
+        <taxii_11:Timestamp_Label>2021-10-20T06:43:08.853465+00:00</taxii_11:Timestamp_Label>
+    </taxii_11:Content_Block>
+</taxii_11:Poll_Response>
```

Comment:
Please make sure there aren't secrets in this file

Reply:
<img width="1525" alt="Screen Shot 2021-11-07 at 12 42 41" src="https://user-images.githubusercontent.com/78307768/140641681-bca5b1b7-ef89-4e44-8855-49ae6948132e.png">